solidity/random-beacon/hardhat.config.ts

```diff
@@ -6,12 +6,14 @@ import "@nomiclabs/hardhat-waffle"
 import "@nomiclabs/hardhat-ethers"
 import "hardhat-deploy"
 import "@tenderly/hardhat-tenderly"
+import "hardhat-gas-reporter"
+import "hardhat-contract-sizer"
 
 const config: HardhatUserConfig = {
   solidity: {
     compilers: [
       {
-        version: "0.8.5",
+        version: "0.8.6",
       },
```

Comment:
What if we enable optimizer, will it help to reduce the contract size (https://github.com/keep-network/keep-core/pull/2630/commits/2e1ae8b28f68ef1c911f8e8cc4ae78f7a5ec1cbf)?

Reply:
It didn't reduce the contract size, but I added it anyway in https://github.com/keep-network/keep-core/pull/2613/commits/47170880fb7f15d26a42ab2f2575c7d4123ebf29, maybe it will generate more efficient code later.